docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -0,0 +1,368 @@
+---
+title: "Deploying a High Availability Teleport Cluster"
+description: "Deploying a High Availability Teleport Cluster"
+---
+
+When deploying Teleport in production, you should design your deployment to
+ensure that users can continue to access infrastructure during an incident in
+your Teleport cluster. In this guide, we will explain the components of a
+high-availability Teleport deployment.
+
+## Overview
+
+A high-availability Teleport cluster revolves around a group of redundant
+`teleport` processes, each of which runs the Auth Service and Proxy Service,
+plus the infrastructure required to support them. 
+
+This includes:
+
+- **A Layer Four load balancer** to direct traffic from users and services to an
```

Comment:
```suggestion
- **A Layer 4 load balancer** to direct traffic from users and services to an
```

Might just be me, but I'm not sure anyone in the history of the internet has ever talked about OSI Layer Four - it's always `Layer 4` or `L4`. See https://en.wikipedia.org/wiki/OSI_model etc. It's jarring to read the word rather than the number.

I assume this is a deliberate thing as it's done throughout this guide, but it looks very incongruous. We use `layer 4` and `layer 7` everywhere else in the docs so I'd encourage us to do the same here.

Reply:
Using words for numbers under 10 has been beaten into me through years of adhering to style guides. I'll change the layer names to use numerals.